### PR TITLE
#16713220 set up heroku build

### PR DESCRIPTION
**Pull request by akinyeleolat**
#### What does this PR do?

- These set up for Heroku deployment

#### Description of task to be completed?

- npm install dependencies
- setup app.js
- setup heroku procFile

#### How should this be manually tested?

  - $ git clone ```https://github.com/andela/king-kong-ah-frontend.git```
  - $ npm install
  - $ npm run build
  - $ node app.js
#### Any background context you want to provide?

- You would need npm to run this application
- you can check the default/initial deployment here ```https://king-kong-ah-frontend-staging.herokuapp.com```

#### What are the relevant pivotal tracker stories?

- PT Story link - [#16713220](https://www.pivotaltracker.com/story/show/16713220)

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web: node app.js
```
Comment (benfluleck) [inline Procfile]: Space needed here

File: app.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "env": {},
+    "formation": {
+      "web": {
+        "quantity": 1
+      }
+    },
+    "addons": [],
+    "buildpacks": [],
+    "stack": "heroku-18"
+  }
```
Comment (benfluleck) [inline app.json]: Space needed here

File: package.json [hunk 3]
```diff
@@ -92,6 +93,7 @@
     "webpack-merge": "^4.2.1"
   },
   "dependencies": {
+    "cors": "^2.8.5",
```
Comment (benfluleck) [inline package.json]: why is this a server ?

**Review by EmmanuelSage — APPROVED** [review]

**Review by frankly034 — APPROVED** [review]

**Review by Sojisoyoye — APPROVED** [review]

**Review by benfluleck — CHANGES_REQUESTED** [review]
Where is my deployment branch to test features

**Pull request merged**